assets/src/components/AssignmentTable.js

```diff
@@ -92,16 +104,27 @@ function AssignmentTable (props) {
     handleInputBlur
   } = props
 
-  const [popoverEl, setPopoverEl] = useState({ popoverId: null, anchorEl: null })
+  const assignmentStatus = {
+    GRADED: 'Graded',
+    SUBMITTED: 'Not Yet Graded',
+    UNSUBMITTED: 'Unsubmitted'
+  }
+
+  const assignmentStatusNames = [assignmentStatus.GRADED, assignmentStatus.SUBMITTED, assignmentStatus.UNSUBMITTED]
```

Comment:
This can be simplified as `Object.values(assignmentStatus)`, this will render as a list
<img width="644" alt="Screen Shot 2020-12-09 at 12 02 51 PM" src="https://user-images.githubusercontent.com/8579775/101661608-803b6600-3a16-11eb-9174-64adea8651ac.png">